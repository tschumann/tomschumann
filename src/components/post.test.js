import React from 'react';
import renderer from 'react-test-renderer';
import Post from './post';

test('test component renders', () => {
	const component = renderer.create(<Post title="Title" body="Body" time="2021-12-31 11:08:00" key="1"/>);
	const tree = component.toJSON();

	expect(tree).toMatchSnapshot();
});

