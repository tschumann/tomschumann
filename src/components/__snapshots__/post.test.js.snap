// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test component renders 1`] = `
<article>
  <h3>
    Title
  </h3>
  <div
    className="post-time"
  >
    Posted at 
    <time
      dateTime="2021-12-31 11:08:00"
    >
      2021-12-31T01:08:00.000Z
    </time>
  </div>
  <div
    className="post-content"
    dangerouslySetInnerHTML={
      {
        "__html": "Body",
      }
    }
  />
</article>
`;
